mmv1/third_party/terraform/services/compute/data_source_google_compute_subnetworks.go

```diff
@@ -57,6 +57,10 @@ func DataSourceGoogleComputeSubnetworks() *schema.Resource {
 							Type:     schema.TypeBool,
 							Computed: true,
 						},
+						"purpose": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
```

Comment:
This change seems unrelated to the purpose of the PR; I'm happy to include it in this PR if you prefer, but it's not necessary. Possibly it was something added to resolve the test failures because they were checking values on the data source instead of testing the resource directly?